core/ui/ViewToolbar/close.tid

```diff
@@ -4,7 +4,10 @@ caption: {{$:/core/images/close-button}} {{$:/language/Buttons/Close/Caption}}
 description: {{$:/language/Buttons/Close/Hint}}
 
 \whitespace trim
-<$button message="tm-close-tiddler" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class=<<tv-config-toolbar-class>>>
+\define focus-selector()
+[data-tiddler-title="$(cssEscapedTitle)$"].tc-tiddler-frame input
+\end
+<$button actions="""<$list filter="[list[$:/StoryList]after<currentTiddler>]"><$set name="cssEscapedTitle" value={{{ [<currentTiddler>escapecss[]] }}}><$action-sendmessage $message="tm-focus-selector" $param=<<focus-selector>>/></$set></$list><$action-sendmessage $message="tm-close-tiddler"/>""" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class=<<tv-config-toolbar-class>>>
```

Comment:
Likewise, it would be useful to refactor the actions string into a macro definition.

Reply:
This is also more readable now